x-pack/plugins/security_solution/public/detections/components/callouts/ml_job_compatibility_callout/translations.tsx

```diff
@@ -13,28 +13,27 @@ import { MlJobCompatibilityLink } from '../../../../common/components/links_to_d
 export const ML_JOB_COMPATIBILITY_CALLOUT_TITLE = i18n.translate(
   'xpack.securitySolution.detectionEngine.mlJobCompatibilityCallout.messageTitle',
   {
-    defaultMessage: 'Your ML jobs may be incompatible with your data sources and/or ML rules',
+    defaultMessage:
+      'New updates are available for your ML jobs, please see documentation before updating Detection Rules to ensure continued coverage',
```

Comment:
Possible headline change. I wanted to emphasize "before you proceed, do this."
```suggestion
      'Before you update your Elastic prebuilt detection rules, check your machine learning rules to ensure continued coverage',
```

Reply:
Since we're in a modal now this title ended up being a little long so I [shortened it](https://github.com/elastic/kibana/pull/128334/files#diff-bfb1a4fcf236f6e864bdc1305e01021f719f724aa61ed5d7fccd557489f3579bR16) to:

> ML rule updates may override your existing rules

Does this work okay?

Reply:
@spong Sorry I missed this earlier -- shortened title looks good!